src/lib/stitching/causality/stitching.ts

```diff
@@ -24,37 +28,15 @@ const resolveLotCentsFieldToMoney = (centsField) => {
 export const causalityStitchingEnvironment = ({
   causalitySchema,
   localSchema,
+  version,
 }: {
   causalitySchema: GraphQLSchema & { transforms: any }
   localSchema: GraphQLSchema
+  version: number
 }) => {
   return {
     extensionSchema: gql`
-      type Lot {
-        internalID: String
-        lot: AuctionsLotState!
-        saleArtwork: SaleArtwork
-      }
-
-      # A connection to a list of items.
-      type LotConnection {
-        # A list of edges.
-        edges: [LotEdge]
-        pageCursors: PageCursors!
-
-        # Information to aid in pagination.
-        pageInfo: PageInfo!
-        totalCount: Int
-      }
-
-      # An edge in a connection.
-      type LotEdge {
-        # A cursor for use in pagination
-        cursor: String!
-
-        # The item at the end of the edge
-        node: Lot
-      }
+      ${version === 2 ? stitchingLotExtensionSchema : ""}
```

Comment:
We don't need to worry about v2 / v1 stuff if we're only writing code for v2 schema

Reply:
This almost definitely causes issues if it is left off because we will otherwise be extending/resolving types that do not exist on v1.

Reply:
Since we're not using V1 schema anymore, this shouldn't matter. Check out stitching for [convection](https://github.com/artsy/metaphysics/tree/master/src/lib/stitching/convection) or [gravity](https://github.com/artsy/metaphysics/tree/master/src/lib/stitching/gravity) as an example.